lib/components/UsersPoolVotesCard.jsx

```diff
@@ -82,7 +71,43 @@ export const UsersPoolVotesCard = (props) => {
           </div>
         </div>
 
-        <div className='flex w-full xs:w-1/2'>
+        {((tokenHolder.hasDelegated && !tokenHolder.selfDelegated) ||
```

Comment:
These would be best collapsed into new vars before the `return <></>`. For instance:
`const holderDelegatedToOthers = tokenHolder.hasDelegated && !tokenHolder.selfDelegated` and `const holderHasBeenDelegatedTo = tokenHolder.canVote && votingPower !== usersPoolBalance` or something along those lines. Whatever makes it easier to read when used in the JSX template.